src/Dfe.PlanTech.Domain/Content/Models/Options/TextRendererOptions.cs

```diff
@@ -15,7 +15,7 @@ public TextRendererOptions(ILogger<TextRendererOptions> logger, List<MarkOption>
 
     public MarkOption? GetMatchingOptionForMark(RichTextMark mark)
     {
-        var matchingOption = _markOptions.FirstOrDefault(option => option.Mark == mark.Type);
+        var matchingOption = _markOptions.Find(option => option.Mark == mark.Type);
```

Comment:
Interesting you've succumbed to the peer pressure of SonarCloud to change this one 😄

Reply:
I applied all fixes indiscriminately.

Apart from the ones that would require a fair bit more work than just a few line changes.

Indiscriminately!